### ISSUE TITLE
Automatic publicPath is not supported in this browser

### ISSUE THREAD

**Issue by letoast** [StaticJsCMS/static-cms#457]
When trying to import the CMS in Sveltekit:
https://stackblitz.com/edit/sveltejs-kit-template-default-hhgway?file=src%2Froutes%2F%2Bpage.svelte
(give it some time and check the console)

I also tried it with a dynamic import inside onMount (to import it client-only)
```js
const CMS = (await import('@staticcms/core')).default;
CMS.init({ config });
```

On Stackblitz the above test didn't do anything (probably didn't even import it), but on my local dev environment it produced the same error as in the title

**Comment by KaneFreeman** [comment]
This should be resolved with 1.2.6.

Also since you are not using react, you will need to use `@staticcms/app` instead of `@staticcms/core`.

**Comment by letoast** [comment]
> This should be resolved with 1.2.6.
> 
> Also since you are not using react, you will need to use `@staticcms/app` instead of `@staticcms/core`.

Woah, you're fast. Great job and thank you! 🙏🙏
